unified_targets/configs/IFF4_TWIN_G.config

```diff
@@ -93,7 +93,9 @@ serial 1 64 115200 57600 0 115200
 # master
 set gyro_to_use = BOTH
 set mag_hardware = NONE
-set baro_hardware = NONE
+set baro_hardware = AUTO
```

Comment:
There is no need to explicitly specify `set baro_hardware = AUTO` - this is the default.

Reply:
But if I remove the "set baro_hardware = AUTO" this line, the baro detect function is not working. So I have to add it for users.

Reply:
Just tested with a STM32F411 target:

```
defaults
```

then

```
# set baro_hardware
baro_hardware = AUTO
```